data/json/furniture_and_terrain/terrain-flora.json

```
@@ -1811,6 +1811,79 @@
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_beehive_natural",
+    "name": "bee hive",
+    "description": "A hive of bees located in a cavity inside a tree, swarming in and out at a feverish pace.  There's probably honey inside.",
+    "symbol": "h",
+    "color": [ "yellow" ],
+    "move_cost": 0,
+    "coverage": 10,
+    "flags": [ "FLAMMABLE_ASH", "NOITEM" ],
+    "examine_action": "harvest_ter",
+    "harvest_by_season": [ { "seasons": [ "spring", "summer" ], "id": "beehive_harv" } ],
+    "transforms_into": "t_tree",
+    "emissions": [ "emit_bees" ],
+    "bash": {
+      "str_min": 2,
+      "str_max": 5,
+      "sound": "crunch!",
+      "sound_fail": "whack!",
+      "ter_set": "t_tree",
+      "items": [ { "item": "wax", "count": [ 10, 25 ] }, { "item": "honeycomb", "count": [ 3, 10 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_apiary",
+    "name": "apiary",
+    "description": "A man-made wooden box designed to hold honeybees and facilitate the care for a hive.  The modular construction allows for inspecting the bees and harvesting their bounty without destroying the hive completely.  This year's harvest should be ready by now.",
+    "symbol": "h",
+    "color": [ "yellow" ],
+    "move_cost": 1,
+    "coverage": 50,
+    "flags": [ "FLAMMABLE_ASH", "NOITEM" ],
+    "examine_action": "harvest_ter",
+    "//": "While this does give a queen bee, handwave it as an exrta to facilitate the development of new hives.",
+    "harvest_by_season": [ { "seasons": [ "summer" ], "id": "beehive_harv" } ],
```

```suggestion
    "harvest_by_season": [ { "seasons": [ "autumn" ], "id": "beehive_harv" } ],
```
Unlike the bee trees these are not established colonies, so you have to wait for your honey.